### ISSUE TITLE
Installation issues with dependencies

### ISSUE THREAD

**Issue by y2j44** [MarketSquare/robotframework-browser#455]
Hey All:

Apologies if i I'm asking this question in the wrong forum but yesterday was my first attempt at installing the browser library and i had some errors installing the dependencies.  I installed the latest node js on a win10 machine, then pip installed the browser library. both successful. I then tried to do rfbrowser init and got some errors below. I ended trying to run the example robot script anyway and got a port error. Any help would be appreciated. thanks
 
Error in library 'Browser': Calling method '_close' of listener 'Browser' failed: Could not connect to the playwright process at port 56346.

npm ERR! code ELIFECYCLE

npm ERR! errno 1

npm ERR! grpc@1.24.3 install: `node-pre-gyp install --fallback-to-build --library=static_library`

npm ERR! Exit status 1

npm ERR!

npm ERR! Failed at the grpc@1.24.3 install script.

npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

**Comment by aaltat** [comment]
Would you have the errors from `rfbrowser init` still available?

**Comment by s-galante** [comment]
I am seeing the same exact issue on Windows 10 and did not this in the Slack channel.  I can give up the log files for the init as well if you can tell me which you are after.  After the run the console shows:
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\y13927\AppData\Roaming\npm-cache\_logs\2020-10-28T14_26_59_064Z-debug.log

If other logs are needed let me know
[2020-10-28T14_26_59_064Z-debug.log](https://github.com/MarketSquare/robotframework-browser/files/5452491/2020-10-28T14_26_59_064Z-debug.log)

**Comment by mkorpela** [comment]
Could be related https://github.com/grpc/grpc-node/issues/1470

**Comment by s-galante** [comment]
I captured the output from the npm grpc install for both default and from source and neither seem to have the 500 download issue noted in that ticket.  The log for the installs are here:
[grpc_install_from_source.log](https://github.com/MarketSquare/robotframework-browser/files/5453751/grpc_install_from_source.log)
[grpc_install_log.zip](https://github.com/MarketSquare/robotframework-browser/files/5453755/grpc_install_log.zip)

Had to zip up the default install, log it 43MB

**Comment by mkorpela** [comment]
Tldr: We have to migrate from using on node side grpc (with native code) to grpc-js

**Comment by mkorpela** [comment]
The specified issues seem to be fixed after grpc to grpc.js change. I'm closing this now.
If (and when) other installation issues happen, please make new issues.

**Comment by mkorpela** [comment]
https://github.com/MarketSquare/robotframework-browser/pull/458

**Comment by mkorpela** [comment]
@s-galante and @y2j44 if it is ok for you, I would like to add you to contributors as this issue is very important one.

**Comment by s-galante** [comment]
@mkorpela I am okay with that as we will need this in a Windows capacity moving forward as well

**Comment by mkorpela** [comment]
@all-contributors please add @s-galante for bug
